docs/en/administrator-guide/config/be_config.md

```diff
@@ -77,6 +77,12 @@ Sometimes the query fails and an error message of `The server is overcrowded` wi
 
 Since this is a brpc configuration, users can also modify this parameter directly during operation. Modify by visiting `http://be_host:brpc_port/flags`.
 
+### brpc_num_threads
+
+This configuration is mainly used to modify the num of bthreads for brpc. The default value is set to -1, which means the num of bthreads is #cpu-cores.
+
+User can set this configuration to a larger value to get better QPS performance.
```

Comment:
Better give an suggestion value or an example. eg: "In our env, we have xx BE with xx core, set this to xx, it can reach xx QPS"

Reply:
Actually, I don't get the real QPS for brpc, I only test stream load operation in our enviroment with  brpc_num_threads set to be 500,  and the relationship between QPS and num of bthreads, I think https://github.com/apache/incubator-brpc/blob/master/docs/cn/benchmark.md is more convincing